### PR TITLE
adding new variables for redhat hybrid cloud console to metrics-utility

### PR DESCRIPTION
**Pull request by aknochow**
##### SUMMARY

Adding variables for shipping metrics to Red Hat Hybrid Cloud Console.  Also adding a secret for the existing gather cronjob to be able to use cloud storage in a future update.

##### ISSUE TYPE

 - New or Enhanced Feature

### CODE REVIEW

File: config/crd/bases/awx.ansible.com_awxs.yaml [hunk 1]
```diff
@@ -1940,25 +1940,31 @@ spec:
                 description: Metrics-Utility Image PullPolicy
                 type: string
               metrics_utility_configmap:
```
Comment (zkayyali812) [inline config/crd/bases/awx.ansible.com_awxs.yaml:1942]: Does `metrics_utility_console_enabled` parameter need to be defined in the CRD as well?

Comment (aknochow) [reply config/crd/bases/awx.ansible.com_awxs.yaml:1942]: Yes and the csv as well. Fixed :)

File: roles/installer/tasks/enable_metrics_utility.yml [hunk 2]
```diff
@@ -1,23 +1,42 @@
...
-- name: Create Kubernetes CronJobs for metrics-utility
+- name: Create metrics-utility Kubernetes CronJob for Red Hat Hybrid Cloud Console
   kubernetes.core.k8s:
-    definition: "{{ lookup('template', item) }}"
+    definition: "{{ lookup('template', item.template) }}"
     apply: true
     wait: true
+  vars:
+    cronjob_name: "{{ item.name }}"
+    metrics_utility_ship_target: crc  # TODO - Update to console when changed
```
Comment (rooftopcellist) [inline roles/installer/tasks/enable_metrics_utility.yml:39]: @aknochow Is this a TODO that needs to be dealt with in this PR? or a reminder for the future?

Comment (aknochow) [reply roles/installer/tasks/enable_metrics_utility.yml:39]: Reminder for the future.  crc will become console in a future version of metrics-utility.

**Review by rooftopcellist — APPROVED** [review]

**Pull request merged**